homework2/task3.c

```diff
@@ -0,0 +1,207 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+#include <stdbool.h>
+
+#define arrElementType int
+
+// #define DEBUG 1
+
+void swap(arrElementType *a, arrElementType *b)
+{
+    arrElementType tmp = *a;
+    *a = *b;
+    *b = tmp;
+}
+
+void bubbleSort(size_t len, arrElementType *arr)
+{
+    for (size_t i = 0; i < len; ++i)
+    {
+        for (size_t j = i + 1; j < len; ++j)
+        {
+            if (arr[i] > arr[j])
+            {
+                swap(&arr[i], &arr[j]);
+            }
+        }
+    }
+}
+
+int countSort(size_t len, arrElementType *arr)
```

Comment:
```suggestion
int countingSort(size_t len, arrElementType *arr)
```
Вот такое название у сортировки